src/index.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
+import React from "react";
+import ReactDOM from "react-dom";
```

Comment:
Please change this line to `import ReactDOM from "react-dom/client";` to clear up the error in your console.